backend/app/controllers/users.js

```diff
@@ -900,10 +900,9 @@ module.exports = {
             } else {
                 updateObj = _.pick(req.body, User.editCols);
             }
-
-            return yield thunkQuery(User.update(updateObj).where(User.id.equals(req.user.id)));
-
-        }).then(function () {
+            yield thunkQuery(User.update(updateObj).where(User.id.equals(req.user.id)));
```

Comment:
Might also be good to return what was inserted/updated. This seems like it will return the success message, whether or not the update was successful. 

Maybe something like:
```
var updatedData = yield thunkQuery(User.update(updateObj).where(User.id.equals(req.user.id)));
if(updatedData) {
   return {
     "message": Successfully inserted data
     "data": updatedData
   }
}
```

Reply:
I think it actually returns an error if the update isn't successful. But that's fine. If we want to return the updated data however, we should probably be consistent on other endpoints.